datadog_lambda/wrapper.py

```diff
@@ -190,6 +193,16 @@ def _after(self, event, context):
             status_code = extract_http_status_code_tag(self.trigger_tags, self.response)
             if status_code:
                 self.trigger_tags["http.status_code"] = status_code
+                if len(status_code) == 3 and status_code.startswith("5"):
+                    submit_errors_metric(context)
+                    if self.span:
+                        self.span.set_traceback()
+                        self.span.error = 1
+                        self.span.set_tags({
+                            ERROR_TYPE: "5xx Server Errors",
```

Comment:
Hey Brett,

I am new to the team so might not be 100% correct, but I think we want to mark 5xx responses as "Error" even though the program does not throw any exception. The intended result UI I think looks like the first graph. So that's why I'm setting the error.type and error.msg here (according the 2nd picture from the datadog doc).
![image](https://user-images.githubusercontent.com/47579703/175392517-9e9ef1c7-724c-42ea-8ffc-c0d26853294b.png)

<img width="868" alt="image" src="https://user-images.githubusercontent.com/47579703/175392182-909fa537-b1fa-4c4a-9a96-cfa9d64a52c9.png">


> Do we also tag this span with the http.status_code ?
Yes, we do here right before the logic I added: https://github.com/DataDog/datadog-lambda-python/blob/d02855ad1833e6a3157908c5d8c00d6afa6a9e36/datadog_lambda/wrapper.py#L195-L195

Reply:
Typically when we mark a span as an error because of the status code we don't set a traceback (e.g. don't we don't call `span.set_traceback()` because there isn't an exception/traceback associated with the "failure").

it is ok to only set `span.error = 1`, as long as we have the `http.status_code` tag also set on the span (thanks for clarifying, I couldn't tell if `trigger_tags` get added to the span or not) then the UI knows what to do with it.

But this is probably more of a product decision, do we want to add a traceback for these specific 500 responses? Do they provide value for customers?

Reply:
(also 👋🏻 welcome!!)

Reply:
Regarding `trigger_tags`, can't be 100% sure but I think we did set it here: https://github.com/DataDog/datadog-lambda-python/blob/a7cb6ef0cd61953d170c50cdbbb78d7a0958c615/datadog_lambda/wrapper.py#L173-L181

I will check with the team about the traceback. Thanks for the comment.

Reply:
Update: we're only setting `self.error=1` and not setting the `error.type` and `error.msg` now. Frontend will handle like what APM does. The PR description is updated to reflect the change and screenshots are also updated.